wails-frontend/src/events/events.tsx

```diff
@@ -1,6 +1,6 @@
 export type promptResult = {
   Success: boolean;
-  Data: any;
+  Data: [];
```

Comment:
why do you assume Data is an array ? it is not the case, for now its a string. please let "any" for now. if the linter complais you can disable it for this file!

Reply:
Ok no problem !

Reply:
Done @peterjah